viz/tap/pkg/util.go

```diff
@@ -0,0 +1,184 @@
+package pkg
```

Comment:
Should this be `util` instead? referencing this as `viz/tap/pkg` was really confusing as `pkg` is this higher level packages directory in `golang`

Reply:
I feel like `pkg` is okay. I see `util` naming kind of equivalent; they are both packages that have general helpers that are imported by other packages.

Reply:
I split these into some better named files but still within `tap/pkg` package. Hopefully that helps in some way.